tests/savepoint/test_translate.py

```diff
@@ -489,6 +488,8 @@ def _subtest(failure_list, subtests, **kwargs):
 def save_netcdf(
     testobj, inputs_list, output_list, ref_data, failing_names, out_filename
 ):
+    import xarray as xr
```

Comment:
excellent, only import it for writing netcdf files, only one all the gt4py work is done